src/test/java/com/laytonsmith/core/functions/DataHandlingTest.java

```diff
@@ -70,8 +70,9 @@ public void testCallProcIsProc() throws Exception {
 				+ " proc('_proc',"
 				+ "     msg('hello world')"
 				+ " )"
-				+ " msg(is_proc('_proc'))"
-				+ " call_proc('_proc')"
+				+ "@name_proc = '_proc'"
+				+ " msg(is_proc(@name_proc))"
```

Comment:
There is a function called dyn() which is meant for use in these cases, as eventually, once more static analysis is complete, this will also cause the same warning. https://methodscript.com/docs/3.3.4/API/functions/dyn.html This forces the compiler to treat the value as dynamic, even if it's hardcoded, and is the preferred solution in this case, and in the eval case as well.